mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/AbstractGenericMojo.java

```diff
@@ -80,34 +80,26 @@
   @Parameter(defaultValue = "${projectBuildDirectory}")
   protected String projectBuildDirectory;
 
-  // mule-packager
-
-  // @Parameter(readonly = true, required = true, defaultValue = "${project.remoteArtifactRepositories}")
-  // protected List<ArtifactRepository> remoteArtifactRepositories;
+  /**
+   * Name of the generated Mule App.
+   *
+   * @parameter alias="appName" default-value="${project.build.finalName}"
```

Comment:
this two annotations shouldn't be needed anymore

Reply:
Done!